lib/ecto/adapters/postgres/connection.ex

```diff
@@ -152,12 +152,12 @@ if Code.ensure_loaded?(Postgrex) do
     end
 
     @impl true
-    def insert(prefix, table, header, rows, on_conflict, returning) do
+    def insert(prefix, table, header, rows, on_conflict, returning, counter_start \\ 1) do
```

Comment:
I think we don't need the `\\`. We should remove it and make sure everything passes it as necessary. :)